GestureHandler.js

```diff
@@ -112,9 +114,15 @@ function transformIntoHandlerTags(handlerIDs) {
   if (!Array.isArray(handlerIDs)) {
     handlerIDs = [handlerIDs];
   }
+
   // converts handler string IDs into their numeric tags
   return handlerIDs
-    .map(handlerID => handlerIDToTag[handlerID] || -1)
+    .map(
+      handlerID =>
+        typeof handlerID === 'string'
```

Comment:
Can we do sth less strict here like for example:
```
handlerID => handlerIDToTag[handlerID] || (handlerID.current && handlerID.current._handlerTag) || -1
```

Also to avoid using typeof